components/feature/readerview/src/main/assets/extensions/readerview/readerview.js

```diff
@@ -5,9 +5,24 @@
 /* Avoid adding ID selector rules in this style sheet, since they could
  * inadvertently match elements in the article content. */
 
+const supportedProtocols = ["http:", "https:"];
+const blockedHosts = ["amazon.com", "github.com", "mail.google.com", "pinterest.com", "reddit.com", "twitter.com", "youtube.com"];
```

Comment:
Reviewing this, it seems like a random selection of choices with no explanation of why we're doing this. Maybe a comment to say this a Fennec list of approved blacklist or something of that sort?

Reply:
K, I will add some comment where this comes from.